src/components/Dashboard/DashboardSection.vue

```diff
@@ -100,6 +100,7 @@ const isMobile = useIsMobile()
 	&__title {
 		font-size: 1.25rem;
 		font-weight: bold;
+		word-break: break-all;
```

Comment:
```suggestion
		word-break: break-word;
```
all | word
-- | --
<img width="86" height="71" alt="Image" src="https://github.com/user-attachments/assets/ce7a930a-a295-4710-bf7b-1637ea7f5d70" /> | <img width="81" height="69" alt="Image" src="https://github.com/user-attachments/assets/4b69c9fb-d73b-49c7-b782-f137872728cf" />